### ISSUE TITLE
How to use a trained network to reconstruct data？

### ISSUE THREAD

**Issue by zuiyr** [gher-ulg/DINCAE#18]
&ensp;&ensp;Hello, after reading your paper and code，I tried using DINCAE1.0 to reconstruct remote sensing data, and so far I have successfully trained and saved the 500th and 1000th training models, which are some network parameters, such as **.ckpt** files.
&ensp;&ensp;I have a question, as mentioned in the paper, the data is divided into **training data** and **testing data**. During the training process, additional Gaussian noise and other cloud masks are added to the training data, while nothing extra is added to the testing data. In addition, 50 images were used for **validation data**, but these 50 images did not participate in model training.So according to my understanding, should we use the trained model parameter file **(. ckpt)** to reconstruct these 50 validation data after training?
&ensp;&ensp;My current approach is to add this line of code in the **reconstruct** function of the code: **save. restore (sess,'E:/DINCAE/DINCAE master/model/model-1000. ckpt ')**, which is added before **"loop over epochs"**.I will create inputdata from 50 validation data and then run DINCAE. At this point, can I understand it as using the trained network model parameters to reconstruct the validation data?I only saw the functions for training the network in your code, and did not find how to directly reconstruct the validation data using the trained network parameters to verify the effectiveness of the model.
&ensp;&ensp;In summary, **my problem is that I don't know how to use the already trained. ckpt parameter files**. My current approach is to load these parameter files, then run DINCAE, and use the **1/1000 epoch output file** as the reconstructed image.The reason why I did this is that the code outputs the reconstruction results of the test data, and the test data does not add anything extra. Therefore, my input data is 50 validation data, and at this point, run DINCAE stores the reconstruction results of the validation images.
&ensp;&ensp;Sorry, I am a beginner and not very familiar with deep learning. I have said too much here. **I sincerely request and hope for your answer. Thank you very much！**
